### ISSUE TITLE
License: can you please add a license to your project?

### ISSUE THREAD

**Issue by orchestr7** [cvssjs/cvssjs.github.io#3]
Hi! I wanted to use your project, but in your codebase the license is missing. Can you please add it to github repo?

**Comment by ericfitz** [comment]
Yes please, the lack of a license and the presence of a copyright notice means that many people can't use your project.
